cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -672,22 +672,31 @@ func main() {
 		debug = log.New(os.Stderr, "", log.LstdFlags)
 	}
 
-	client := retryablehttp.NewClient()
-	client.Logger = debug
+	var sg Sourcegraph
+	if rootURL.IsAbs() {
+		client := retryablehttp.NewClient()
+		client.Logger = debug
+		sg = &sourcegraphClient{
+			Root:     rootURL,
+			Client:   client,
+			Hostname: *hostname,
+		}
+	} else {
```

Comment:
update the help text for `-sourcegraph_url` to indicated that it supports directories